### ISSUE TITLE
Remove duplicate pre...

### ISSUE THREAD

**Issue by kobeaerts** [ryssbowh/craft-prefetch#1]
Hey, really loving this little Craft plugin.

I've noticed that when I do multiple
```
{% do craft.prefetch.preload('url') %}
{% do craft.prefetch.preload('url') %}
{% do craft.prefetch.preload('url') %}
```

It outputs multiple preload tags, is it possible to reduce this to 1 preload tag?

**Comment by ryssbowh** [comment]
Hi,

I've never seen a preload with multiple urls, is that even possible ?
Each preload will have different arguments also, such as "as", so I don't think this is possible (?)

**Comment by kobeaerts** [comment]
I'm sorry, I explained it wrong.

This input:
```
{% do craft.prefetch.preload('https://www.google.com') %}
{% do craft.prefetch.preload('https://www.google.com') %}
{% do craft.prefetch.preload('https://www.google.com') %}
```
outputs:
```
<link href="https://www.google.com" rel="preload" as="script" />
<link href="https://www.google.com" rel="preload" as="script" />
<link href="https://www.google.com" rel="preload" as="script" />
```
While the script should only be preloaded once like this:
```
<link href="https://www.google.com" rel="preload" as="script" />
```

**Comment by ryssbowh** [comment]
Right, I see
Will fix it.

**Comment by ryssbowh** [comment]
This is fixed in 1.0.4, thanks for reporting

**Comment by kobeaerts** [comment]
Wonderful! 😎
